ffcx/codegeneration/C/form.py

```diff
@@ -87,7 +87,7 @@ def generator(ir: FormIR, options):
     integral_ids = []
     integral_offsets = [0]
     # Note: the order of this list is defined by the enum ufcx_integral_type in ufcx.h
-    for itg_type in ("cell", "exterior_facet", "interior_facet"):
+    for itg_type in ("cell", "exterior_facet", "interior_facet", "ridge"):
```

Comment:
Are there problems where you may want to only include ridges on the boundary? If so, we should include both `exterior_ridge` and `interior_ridge` here.

Reply:
I think this is a multi-step approach. lets start with exterior ridges. 
Interior ridges have the issue that the restriction in `ufl` is not really sufficient (an edge can be connected to `n` cells, thus +, - is not sufficient).

Reply:
Ah yes, I was mixing up +/- and boundary/internal. Ignore my comment